docs/Features.md

```diff
@@ -1,4 +1,4 @@
-This is a list of features implemented since the [original Theos](https://github.com/theos/theos/tree/legacy) by Dustin Howett. This list is sorted by the date the feature was added. The latest additions are at the bottom.
```

Comment:
In general this whole file needs to be ditched, we haven’t updated it in some time (intentionally because it’s a pretty messy/vague way to document things), but these quick edits are at least a good start for the moment.

Reply:
Alright. I added on to the "To do" of that file, noting how it should be removed once its contents have been properly documented.